index.html

```diff
@@ -0,0 +1,1361 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+    crossorigin="anonymous" />
+  <link rel="shortcut icon" href="/image/logo.jpg" type="image/x-icon" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css" />
+
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;700&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="css/style.css" />
+  <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
+</head>
+
+<body>
+  <!-- ++++++++++++++++++++++++++++++++++++++++++++++++++
+    Navigation 
+    ++++++++++++++++++++++++++++++++++++++++++++++++++ -->
```

Comment:
Please remove all comments from your HTML and CSS code:
![image](https://user-images.githubusercontent.com/34653764/84459464-6e435b00-ac5f-11ea-845a-88992d2d74fb.png)